fdbserver/DDTeamCollection.actor.cpp

```diff
@@ -232,12 +232,17 @@ class DDTeamCollectionImpl {
 
 			std::vector<Reference<TCTeamInfo>> candidateTeams;
 			int unhealthyTeamCount = 0;
+			int notEligibileTeamCount = 0;
```

Comment:
Add in PR summary why not using notEligibileTeamCount leads to TraceTooManyLines